### DISCUSSION TITLE
Deploy Typescipt Apollo Graphql API to vercel

### DISCUSSION THREAD

**Discussion by shimaxu** [vercel/community#404] [Help] [answered]
Hello everyone,

I would like to know how i can deploy Graphql Api made with nexus and prisma to vercel platform. 

I couldn't find any related answer to this. but there are fewer solutions that i found in vercel youtube channel using NextJS. Also there's bunch of youtube videos in prisma channel. i couldn't find any similar implementation for my graphql API that i made with the help of how to graphql with typescript-apollo tutorial. https://www.howtographql.com/typescript-apollo/0-introduction/

Note: I am not using any Javascript framework like NextJS, also i am using typescript in this project.

**Comment by samcx** [comment, marked as answer]
I do not not recommend trying to deploy your GraphQL server on Vercel since Vercel is a Serverless and Edge-first platform. 

A GraphQL server, or any complex Node.js server, will be better suited on a dedicated server (e.g. Heroku).
